src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticListener.cs

```diff
@@ -258,7 +258,7 @@ public override bool IsEnabled(string name, object arg1, object arg2 = null)
         /// </summary>
         public override void Write(string name, object value)
```

Comment:
It seems like the name and the value can be null here.

Reply:
yes, I made them all non null because of KeyValuePair constructor, will update